docs/semgrep-code/getting-started.md

```diff
@@ -29,333 +28,146 @@ import PlatformSigninGithub from "/src/components/procedure/_platform-signin-git
 import PlatformSigninGitlab from "/src/components/procedure/_platform-signin-gitlab.md"
 import PlatformDetectGhRepos from "/src/components/procedure/_platform-detect-ghrepos.md"
 
-# Getting started with Semgrep Code
-
-Secure your code quickly and continuously by scanning with Semgrep Code, a fast and lightweight SAST (Static Application Security Testing) engine that leverages Semgrep OSS.
-
-<SemgrepScan />
-
-Semgrep Code is transparent: you can fully configure what rules are run and inspect the Semgrep syntax to understand how the finding was detected. The content of a rule can be customized to improve the true positive rate of a rule or its message to fellow developers.
```

Comment:
Wonder if it's worth it to move this to the Semgrep Code > Overview doc, which is our "how it works" / [explanation](https://diataxis.fr/) document.